src/Symfony/Component/Routing/Tests/Requirement/RequirementTest.php

```diff
@@ -224,10 +224,7 @@ public function testUidBase58KO(string $uid)
     }
 
     /**
-     * @testWith    ["00000000-0000-0000-0000-000000000000"]
-     *              ["ffffffff-ffff-ffff-ffff-ffffffffffff"]
-     *              ["01802c4e-c409-9f07-863c-f025ca7766a0"]
-     *              ["056654ca-0699-4e16-9895-e60afca090d7"]
+     * @dataProvider provideUidRfc4122
```

Comment:
Why changing this?

Reply:
I think it's because he wants to reuse these data sets for a second test.

Reply:
Yes, it's a way to explicitly use the dataset of RFC4122 with the new constant and show results should be the same